src/Compiler/SyntaxTree/SyntaxTree.fs

```diff
@@ -37,7 +37,7 @@ type SynLongIdent =
 
     member this.Range =
         match this with
-        | SynLongIdent([], _, _) -> failwith "rangeOfLidwd"
+        | SynLongIdent([], _, _) -> Range.Zero
```

Comment:
`Range.Zero` is commonly used in similar scenarios in the `SyntaxTree` file. But I agree we could use `option` here

Reply:
> Range.Zero is commonly used in similar scenarios in the SyntaxTree file.

Well, that's wrong because it means some node with a range like `(10,20)-(10,30)` can now contain a node with range `(0, 0)-(0, 0)` which is outside of its parent node range. Anyway, we need to fix this more systematically in future.

Reply:
Done